### PR TITLE
Mobile-menu

### PR DESCRIPTION
**Pull request by ElShaer398**
Mobile menu feature for Portfolio-setup-and-mobile-first project at Microverse.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -4,20 +4,30 @@
...
       <ul class="nav-items">
-        <li>Portoflio</li>
-        <li>About</li>
-        <li>Contact</li>
+        <li class="items">Portoflio</li>
+        <li class="items">About</li>
+        <li class="items">Contact</li>
       </ul>
-      <img src="images/nav.png" alt="nav">
     </nav>
```
Comment (mckent05) [inline index.html:31]: - According to the project requirements, `When the user clicks (or taps) any of the mobile menu options, a correct part of the page is displayed.`. Your project doesn't function as required, please fix this 🙏

**Review by mckent05 — CHANGES_REQUESTED** [review]
Hi @ElShaer398, 

# STATUS: CHANGES REQUIRED ♻️ ♻️ 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
almost there

## Highlights
- All linter checks are passing ✔️ ✔️ 
- Your PR is professional 👍 👍 


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Reem-lab — COMMENTED** [review]
Hi Team, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that doesn't have the file for javascript linters so kindly add the eslint file and update the linters.yml file to have the js linters as well 💯♻🔥 please check this link [HTML CSS JS linter](https://github.com/microverseinc/linters-config/tree/master/html-css-js)

- I can see your README is not complete so kindly complete the missing parts as well to make it completed 🔥♻💯✅

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: index.html [hunk 2]
```diff
@@ -4,20 +4,30 @@
...
 <body>
-  <header>
+  <div class="popUpNav">
+    <div class="mainXmark">
+      <a class="x-mark"><i class="fa-solid fa-xmark"></i></a>
+    </div>
+    <ul class="nav-items-2">
+      <li class="items"><a href="#cards-section-id">Portoflio</a></li>
+      <li class="items"><a href="#about-myself-section-id">About</a></li>
+      <li class="items"><a href="#contact-form-section-id">Contact</a></li>
```
Comment (mikethreels) [inline index.html:19]: - [ ] the colour of the text in the mobile menu should be white or hex #FFFFFF.

**Review by mikethreels — CHANGES_REQUESTED** [review]
Hi Mohammad, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
- linters are passing

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by christianbiring1 — COMMENTED** [review]
Hi @ElShaer398 , 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

# Invalid Code Review Request :heavy_multiplication_x: 
Invalid


You have submitted a project, that has linters errors, kindly fix that and resubmit for another review.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: index.html [hunk 3]
```diff
@@ -4,20 +4,30 @@
...
   <title>setup and mobile first</title>
 </head>
 <body>
-  <header>
+  <div class="popUpNav">
+    <div class="mainXmark">
+      <a class="x-mark"><i class="fa-solid fa-xmark"></i></a>
+    </div>
+    <ul class="nav-items-2">
+      <li class="items"><a href="#cards-section-id">Portoflio</a></li>
```
Comment (danmainah) [inline index.html:17]: - [ ] [optional] Kindly ensure the spelling of the portfolio is correctly done.

**Review by danmainah — APPROVED** [review]
Hi @Team, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 🎉 🎉 

## Highlights
✔️ No linter errors
✔️ Github flow was correctly used
✔️  Good design

## Optional suggestions
_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**